snuba/query/processors/logical/low_cardinality_processor.py

```diff
@@ -41,12 +41,7 @@ def transform_expressions(exp: Expression) -> Expression:
                 ),
             )
 
-        if (
-            state.get_int_config(
-                "use.low.cardinality.processor", settings.USE_CARDINALITY_CASTER
-            )
-            == 0
-        ):
+        if state.get_int_config("use.low.cardinality.processor", 1) == 0:
```

Comment:
Enabling this by default caused some test failures last time that blocked sentry master, have we checked it won't happen again?

Reply:
The processor now works differently, the `ifNull(col, "")` is removed, instead it casts to a `Nullable(String)`. I ran some of the failed tests locally with the new processor and they passed